Expensify/App
# Add a tooltip for profile avatar and online status icon

### Details
Tooltip added for profile avatar and online status/offline dot

### Fixed Issues

$ https://github.com/Expensify/App/issues/5973

### Tests

1. Navgiate to New Expensify
2. Hover over the avatar profile icon and the online dot

### QA Steps

1. Navigate to New Expensify
2. Hover over the avatar profile icon and the online dot
3. Make sure there is a tooltip over the avatar with name of the user
4. And overing over the online dot, you can see tooltip describing your status

### Tested On

### Screenshots

<img width="381" alt="Screen Shot 2021-10-26 at 9 08 06 AM" src="https://user-images.githubusercontent.com/11542415/138809194-dd4bc8db-19fb-4367-a8c7-3c541a596daf.png">
<img width="384" alt="Screen Shot 2021-10-26 at 9 08 02 AM" src="https://user-images.githubusercontent.com/11542415/138809254-07b1398b-12bf-4ec6-80da-4cad6bf8b279.png">
